src/reference/memory.md

```diff
@@ -225,3 +225,9 @@ Moon deinit
 
 Weak references are part of the s package. See the [package documentation](../packages/s/1f4f6.html) to
 learn more about their usage.
+
```

Comment:
I was very sure I had written documentation on 🏮. I need to check that when I’m back at my Mac.

Reply:
When I encountered it in the standard library code, I grepped and googled and couldn't find any mention of it. If anything, you might have an uncommitted draft.

Reply:
Ok, seems like I’ve never documented this after all. Can you please remove that TODO or the whole section? If I merge it’ll go live immediately and that [won’t look too good](https://deploy-preview-57--emojicode.netlify.com/docs/reference/memory.html#detecting-shared-values). I’ll write this up later.